### ISSUE TITLE
Plans to include reporting output to junit/xunit files?

### ISSUE THREAD

**Issue by bricejlin** [DispatchMe/meteor-mocha-phantomjs#22]
A lot of CI runners look for this file format :)

**Comment by aldeed** [comment]
See my suggestion here: https://github.com/DispatchMe/meteor-mocha/issues/2#issuecomment-227936088

**Comment by chriswessells** [comment]
I tried the solution and was unable to get an output file.  I even followed ayhid's hack and was unsuccessful.  I was unable to get any thing to output to a file.

As per https://www.npmjs.com/package/mocha-phantomjs I tried all the reporter formats and found the following:
I was albe to verify the reporter formats output to the console correctly.
dot - client and server
min - server
nyan - server
list - client and server
doc - client and server
teamcity - fail
json - client and server
json-cov - client and server
xunit - client and server
progress - client and server
landing - server
markdown - client and server

I added ayhid's meteor-mocha-core to my packages directory and added another dependency for reporter-file as a dependency after xunit-file.  

I tried setting all the environment variables and was not able to generate a file.  I am in DevOps so getting this information is critical.  I have tried to grep or sed the output, or output to file via > but have not been successful.  If someone could point me to where the in the code I can capture the stdout I can find a way to write it to a file.
